src/result.rs

```diff
@@ -70,71 +75,93 @@ pub enum CqlType {
 #[napi]
 impl QueryResultWrapper {
     pub fn from_query(internal: QueryResult) -> QueryResultWrapper {
-        QueryResultWrapper { internal }
+        let value = match internal.clone().into_rows_result() {
```

Comment:
:wrench: Needless `clone()` - `internal` is an owned `QueryResult` that we can consume.